.eslintrc.json

```diff
@@ -0,0 +1,40 @@
+{
+  "root": true,
+  "ignorePatterns": ["**/*"],
+  "plugins": ["@nx"],
+  "overrides": [
+    {
+      "files": ["*.ts", "*.tsx", "*.js", "*.jsx"],
+      "rules": {
+        "@nx/enforce-module-boundaries": [
```

Comment:
It's generated by Nx this way. It does already check some things without an additional config, e.g. importing between projects only via path aliases, and no circular dependencies

I would like to add more configuration, so that we also include some tag-based rules, but I think it makes sense to way until @BioPhoton's PR for #6 lands (he needs this to be merged first), since he's also adding more projects (e.g. `core` and `cli`), and we'll also need to separate the e2e project #19 (`cli` imports from `plugin-eslint` and `plugin-lighthouse` in test files, which would break our lint rule).

Reply:
I've created an issue for this here: #28